src/main/java/io/github/mountainrange/mule/gameplay/WorldMap.java

```diff
@@ -5,19 +5,22 @@
 import io.github.mountainrange.mule.enums.MuleType;
 import javafx.geometry.Point2D;
 
+import java.awt.*;
+import java.io.Serializable;
 import java.util.*;
+import java.util.List;
 import java.util.stream.Collectors;
 
 /**
  * A class to represent the map and facilitates interactions between the actual data store and the rest of the program.
  */
-public class WorldMap<T extends Tile> implements Iterable<Tile> {
+public class WorldMap<T extends Tile> implements Iterable<Tile>, Serializable {
 
 	/** Backing instance of Grid. */
 	private Grid<T> baseGrid;
 
-	/** List of tiles owned by each player, sorted by the MULE installed on them. */
-	private Map<Player, EnumMap<MuleType, Set<Tile>>> productionTiles;
+	/* List of tiles owned by each player, sorted by the MULE installed on them */
+	private Map<Player, EnumMap<MuleType, Set<Point>>> productionTiles;
```

Comment:
Don't worry, I've been messing with `Serializable` lately. You can override `readObject` and `writeObject`, but you can also do `in.defaultReadObject` and `out.defaultWriteObject` to take advantage of Java automagically serializing instance variables that aren't transient, and then do some initialization afterwards to initialize transient fields. I've had some JavaFX initialization issues that I've been trying to debug, though.